english/forest-canopy/forest-canopy.md

```diff
@@ -1,5 +1,7 @@
 With your fork placed firmly between your clenched jaws you grab for the highest branch.
 
-You smash through the canopy, but are able to grab for something soft and swing from it like a vine. 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.